### PR TITLE
[Feature] Added Banner Section

### PR DESCRIPTION
**Pull request by YuxinZhang214**
### After this PR
- Added Banner into footer section

### Screenshots
- The Banner Section
Screenshot 2023-10-20 at 20 39 30

- The Whole page
Screenshot 2023-10-20 at 20 39 52
Screenshot 2023-10-20 at 20 39 42

### Note
- Tried various layout but decided to structure like this.
- Tried to hide the banner entirely under the screen, but I feel 80% of the content feels the best 
- Haven't decide on the background colour yet.
- Haven't found the method for mapping username to the real name, leave as username now
- Open to any advice :)

### CODE REVIEW

**Comment by YuxinZhang214** [comment]
Update Banner Colour to a customised blue 

Screenshot 2023-10-22 at 21 48 51

**Review by ozgurakgun — COMMENTED** [review]
Looks good, a couple of minor points.

File: components/layout/Banner.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+              <div className="flex items-center p-2">
+                <img src="conjure-cp-logo.png" alt="conjure-cp" className="w-16 h-16" />
+                  <div className="text-xl font-semibold ml-4">
+                    <a href="https://github.com/conjure-cp" target="_blank" rel="noopener noreferrer" className="hover:underline">
+                      conjure-cp
+                    </a>
+                  </div>
+              </div>
+              <div className="mb-2">Workload Planner</div>
+              <div className="mb-2">Version: 1.0</div>
```
[the remark this answers is not in the document]
Comment (YuxinZhang214) [reply components/layout/Banner.js:72]: Yeah! I will make both dynamic. I also found that the banner might be positioned in a different place for the start page (when you don't have a project), i will make a pull request after i fix this.

[the remark this answers is not in the document]
Comment (YuxinZhang214) [reply components/layout/Banner.js:72]: * i will commit to this pull request

**Comment by ozgurakgun** [comment]
I am wondering whether we should we just remove the version. I don't mean just from the banner, from everywhere. There is no need for versioning in a demo...

**Comment by ozgurakgun** [comment]
sorry this is becoming a moving goal post for you @YuxinZhang214! but I really like the banner and just want to simplify what we can in the meantime.

**Comment by YuxinZhang214** [comment]
> sorry this is becoming a moving goal post for you @YuxinZhang214! but I really like the banner and just want to simplify what we can in the meantime.

Sure! I will remove the version from everywhere. Haha, don't worry, I am also expecting every little feedback for the banner and for the updates because i am getting used to the banner and i have no idea how to make it better :) I am glad you like the banner!

**Comment by YuxinZhang214** [comment]
i didn't manage to fix the banner location for the start page. i might need a little from Xu and Daniel in the next meeting.

**Comment by ozgurakgun** [comment]
You can reach out to them and explain the problem if you like? Keep me in the loop as well, I might be able to help too.

Also: I just realised I created a merge conflict for this PR. if you merge main back to your branch you will be able to resolve the conflict.

**Comment by YuxinZhang214** [comment]
> You can reach out to them and explain the problem if you like? Keep me in the loop as well, I might be able to help too.
> 
> Also: I just realised I created a merge conflict for this PR. if you merge main back to your branch you will be able to resolve the conflict.

Yeah, i will email them. I will also merge the main branch back

**Pull request merged**